### PR TITLE
Hail backend - change callset population

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: seqr/utils/search/hail_search_utils.py [hunk 1]
```diff
@@ -28,10 +28,14 @@ def get_hail_variants(samples, search, user, previous_search_results, genome_ver
     end_offset = num_results * page
     search_body = _format_search_body(samples, genome_version, user, end_offset, search)
 
+    frequencies = search_body.pop('freqs', None)
+    if frequencies and frequencies.get('callset'):
+        frequencies['seqr'] = frequencies.pop('callset')
```
Comment (ShifaSZ) [inline seqr/utils/search/hail_search_utils.py:33]: It seems to me that `seqr` here means `seqr callset`. Is it better to use `seqrCallset` or `seqrPop`?

File: ui/shared/components/panel/variants/Frequencies.jsx [hunk 2]
```diff
@@ -157,11 +157,15 @@ const sectionTitle = ({ fieldTitle, section }) => (
...
 const HET_SECTION = 'Heteroplasmy'
 
+const SV_CALLSET_POP = { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE }
 const CALLSET_POP = { field: 'callset', fieldTitle: 'This Callset', acDisplay: 'AC' }
+const SEQR_POP = { ...CALLSET_POP, field: 'seqr', fieldTitle: 'seqr' }
 
 const POPULATIONS = [
-  { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE },
+  SV_CALLSET_POP,
+  { ...SV_CALLSET_POP, field: 'sv_seqr', fieldTitle: 'seqr' },
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/Frequencies.jsx:166]: Same as above for the `fieldTitle` field.

File: ui/shared/components/panel/variants/Frequencies.jsx [hunk 3]
```diff
@@ -157,11 +157,15 @@ const sectionTitle = ({ fieldTitle, section }) => (
 const HOM_SECTION = 'Homoplasmy'
 const HET_SECTION = 'Heteroplasmy'
 
+const SV_CALLSET_POP = { field: 'sv_callset', fieldTitle: 'This Callset', acDisplay: 'AC', helpMessage: SV_CALLSET_CRITERIA_MESSAGE }
 const CALLSET_POP = { field: 'callset', fieldTitle: 'This Callset', acDisplay: 'AC' }
+const SEQR_POP = { ...CALLSET_POP, field: 'seqr', fieldTitle: 'seqr' }
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/Frequencies.jsx:162]: Should the value for the `fieldTitle` be `seqr Callset`?

File: seqr/utils/search/hail_search_utils.py (hunk 1 above)
Comment (hanars) [reply seqr/utils/search/hail_search_utils.py:33]: No, that would be an anti-pattern.  This is nested under a `freqs` dictionary in the body, so its very explicit that its rteferring to a frequency, and we don't  use `gnomad_genomes_pop` or the like for the other populations

File: ui/shared/components/panel/variants/Frequencies.jsx (hunk 2 above)
Comment (hanars) [reply ui/shared/components/panel/variants/Frequencies.jsx:166]: Its very clear in the UI that this is referring to a population frequency. Adding the word "Callset" is really inaccurate or misleading - it implies all the samples in seqr have been called together, which they haven't. And adding the words "population" or "frequency" clutters to UI without adding any information to the user

File: ui/shared/components/panel/variants/Frequencies.jsx (hunk 3 above)
Comment (hanars) [reply ui/shared/components/panel/variants/Frequencies.jsx:162]: see above, but no

File: ui/shared/components/panel/variants/Frequencies.jsx (hunk 2 above)
Comment (ShifaSZ) [reply ui/shared/components/panel/variants/Frequencies.jsx:166]: I see. But the statement `all the samples in seqr have been called together` confuses me. What does the word `called` mean exactly?

**Review by ShifaSZ — APPROVED** [review]

File: ui/shared/components/panel/variants/Frequencies.jsx (hunk 2 above)
Comment (hanars) [reply ui/shared/components/panel/variants/Frequencies.jsx:166]: It means that they were run through a joint calling pipeline, like GATK, which is how a joint called VCF is generated

**Pull request merged**